Yoast/wordpress-seo
# Add yoast prefixes to CSS variables

## Context


* Adds the `yoast`-prefix to all CSS variables

## Summary


This PR can be summarized in the following changelog entry:

* Adds the `yoast`-prefix to all CSS variables

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the latest `develop` from the monorepo
* Make sure all colors, font-sizes and other variables are still correctly set on the admin pages

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-102
